tests/translator/output/error_connector.json

```diff
@@ -38,7 +38,7 @@
       "errorMessage": "Resource with id [MissingSnsTopicArn] is invalid. Destination.Arn is missing."
     },
     {
-      "errorMessage": "Resource with id [BothIdAndOtherProps] is invalid. Must provide either 'Id' or a combination of the other properties, not both."
+      "errorMessage": "Resource with id [BothIdAndOtherProps] is invalid. Must provide either 'Id' (with optional 'Qualifier') or a combination of the other properties, not both."
```

Comment:
The message is kind of hard to read. It reads like the "combination of other properties" cannot include "Qualifier"

Reply:
Tend to agree here. Might be easier to just enumerate the valid parameters that can be combined together. For e.g. "Property 'Arn' is not valid when 'Id' is provided. Properties that can be provided if 'Id' is specified: ['Qualifier']."